sale_multi_template_application/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2024 Tecnativa - Carlos López
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Sale multi template application",
+    "version": "17.0.1.0.0",
+    "summary": """Allows multiple templates to be applied to the same sale order,
+    keeping the existing order lines without deleting the previous ones.""",
```

Comment:
The summary shouldn't be so long, and most of the times, the name is enough (but a significant name):

```suggestion
    "name": "Apply multiple quotation templates",
    "version": "17.0.1.0.0",
```